__tests__/reporters/__snapshots__/console-reporter.js.snap

```diff
@@ -24,14 +24,14 @@ Object {
 exports[`ConsoleReporter.footer 1`] = `
 Object {
   "stderr": "",
-  "stdout": "[2K[1G✨  Done in 0.00s.",
+  "stdout": "[2K[1G✨  Done in 0.00 s.",
 }
 `;
 
 exports[`ConsoleReporter.footer 2`] = `
 Object {
   "stderr": "",
-  "stdout": "[2K[1G✨  Done in 0.00s. Peak memory usage 0.00MB.",
+  "stdout": "[2K[1G✨  Done in 0.00 s. Peak memory usage 0.00 MB.",
```

Comment:
Shouldn't that be MiB instead of MB if we want to comply to SI? (Refs: https://en.wikipedia.org/wiki/Mebibyte)

Reply:
OK, done in the second commit, thanks.